frontend/src/components/common/header/header.tsx

```diff
@@ -1,27 +1,41 @@
+import { useState } from 'react';
+
 import { FC } from 'common/types/types';
 import { Image } from 'components/common/common';
+import { Popup } from './components/components';
 import logo from 'assets/img/logo.svg';
 import defaultUserAvatar from 'assets/img/avatar-default.svg';
-
 import styles from './styles.module.scss';
 
-const Header: FC = () => (
-  <header>
-    <div className={styles.headerWrapper}>
-      <div className={styles.logoWrapper}>
-        <Image width="150" height="94" src={logo} alt="logo" />
-      </div>
-      <div className={styles.userWrapper}>
-        <Image
-          width="50"
-          height="50"
-          src={defaultUserAvatar}
-          alt="user avatar"
-          isCircular
-        />
+const Header: FC = () => {
+  const [isMenuPopupVisible, setIsMenuPopupVisible] = useState<boolean>(false);
+
+  const handleOpenMenuPopup = (): void => setIsMenuPopupVisible(true);
+
+  const handleCloseMenuPopup = (): void => setIsMenuPopupVisible(false);
+
+  return (
+    <header>
+      <div className={styles.headerWrapper}>
+        <div className={styles.logoWrapper}>
+          <Image width="150" height="94" src={logo} alt="logo" />
+        </div>
+        <div className={styles.userWrapper}>
+          <Image
+            width="50"
+            height="50"
+            src={defaultUserAvatar}
+            alt="user avatar"
+            isCircular
+            onClick={handleOpenMenuPopup}
```

Comment:
```suggestion
```
you need to wrap it with button element
onClick should be only on the interactive elements

Reply:
isn't resolved

Reply:
onClick was removed from Image component, it was wrapped with button element and onClick was set to button element. Am I missing something?